mod/likes/views/default/likes/js.php

```diff
@@ -24,4 +25,39 @@
 };
 
 elgg.register_hook_handler('getOptions', 'ui.popup', elgg.ui.likesPopupHandler);
-elgg.ui.registerTogglableMenuItems('likes', 'unlike');
\ No newline at end of file
+
```

Comment:
I figure this is BC because it still works if this view has been overridden, it just won't get this bug fix.